src/relay/qnn/op/concatenate.cc

```diff
@@ -207,6 +207,7 @@ RELAY_REGISTER_OP("qnn.concatenate")
                   "The quantization zero_point of the output tensor.")
     .set_support_level(11)
     .add_type_rel("QnnConcatenate", QnnConcatenateRel)
+    .set_attr<TNonComputational>("TNonComputational", true)
```

Comment:
Where is `TNonComputational` defined? I don't see it in this PR.

Reply:
TNonComputational is defined in include/tvm/relay/op_attr_types.h